### PR TITLE
configcenter readme, import and run files

### PR DESCRIPTION
**Pull request by beiwei30**

### CODE REVIEW

**Review by LaurenceLiZhixin — APPROVED** [review]

File: configcenter/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+### Example
+
+If use Zookeeper as config center, global externalized configuration and application scope configurations will be layout like below:
+
+```
+dubbo
+└── config
+    ├── dubbo
+    │   └── dubbo.properties <- 全局
```
Comment (AlexStocks) [inline configcenter/README.md:25]: pls translate the chinese into english

File: configcenter/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    ├── user-info-client     
+    │   └── dubbo.properties <- 服务消费者
```
Comment (AlexStocks) [inline configcenter/README.md:27]: pls translate the chinese into english

File: configcenter/README.md (hunk 1 above)
Comment (beiwei30) [reply configcenter/README.md]: done.

File: configcenter/README.md (hunk 2 above)
Comment (beiwei30) [reply configcenter/README.md]: done.

**Review by AlexStocks — APPROVED** [review]

File: configcenter/nacos/go-client/cmd/client.go [hunk 3]
```diff
@@ -19,26 +19,26 @@ package main
 
 import (
 	"context"
+	hessian "github.com/apache/dubbo-go-hessian2"
```
Comment (zouyx) [inline configcenter/nacos/go-client/cmd/client.go:22]: move to second block

**Review by zouyx — APPROVED** [review]

**Pull request merged**